### PR TITLE
Add test to hide task list

### PR DESCRIPTION
**Pull request by lanej0**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Adds an e2e test to hide the task list and then ensure that store management is displayed

Closes https://github.com/woocommerce/woocommerce-quality/issues/594
Closes https://github.com/woocommerce/woocommerce-quality/issues/595

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. `pnpm test:e2e activate-and-setup`
2. Ensure CI passes

### CODE REVIEW

File: plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+const { test, expect } = require( '@playwright/test' );
+const { exec } = require( 'child_process' );
+
+test.describe( 'WC Home Task List > ', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.beforeAll( async ( {} ) => {
```
Comment (lanej0) [inline plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js:7]: It seems that hiding the task list writes an option to the wp_options table.  You can delete this using the `wp cli` tool, but external hosts might not have that installed.  So I skipped the test for everything except `wp-env` runs.

Comment (lanej0) [reply plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js:7]: Definitely open to alternatives on how we could handle this reset though.

Comment (adimoldovan) [reply plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js:7]: Is there an API alternative?

Comment (lanej0) [reply plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js:7]: I dug around to see if I could find one, but didn't see anything.

**Comment by rodelgc** [comment]
Hi @lanej0 Seems this PR is out of date with trunk. If you could update it, that would be great. I could take a look after.

File: plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+			} );
+		} );
+	} );
+
+	test(
+		' Can hide the task list',
+		{ tag: [ '@skip-on-default-pressable', '@skip-on-default-wpcom' ] },
+		async ( { page } ) => {
+			await test.step( 'Load the WC Admin page', async () => {
+				await page.goto( '/wp-admin/admin.php?page=wc-admin' );
```
Comment (Stojdza) [inline plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js]: Following changes from [this PR](https://github.com/woocommerce/woocommerce/pull/53228) and supporting multisite setup, URLs should start without `/` or with `./`

**Review by Stojdza — COMMENTED** [review]
Left a comment related to another PR for multisite suppoort

File: plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js (hunk 2 above)
Comment (lanej0) [reply plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js]: URL updated.  Thanks for that @Stojdza

**Review by Stojdza — APPROVED** [review]
LGTM

**Pull request merged**